lib/node_modules/@stdlib/math/base/special/acscd/README.md

```diff
@@ -0,0 +1,117 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# acscd
+
+> Compute the [arccosecant][arccosecant] in degree of a double-precision floating-point number.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var acscd = require( '@stdlib/math/base/special/acscd' );
+```
+
+#### acscd( x )
+
+Computes the [arccosecant][arccosecant] (in degree) of a double-precision floating-point number.
+
+```javascript
+var sqrt = require( '@stdlib/math/base/special/sqrt' );
+var v = acscd( Infinity );
+// returns 0.0
+
+v = acscd( 2 * sqrt(3) / 3 );
+// returns 60.00000000000001
```

Comment:
We usually around to up to three digits after the decimal point and use `~` for approximate values in return annotations:

```suggestion
// returns ~60.0
```